### PR TITLE
Direct people to sign up themselves

### PR DESCRIPTION
**Pull request by wbl**

### CODE REVIEW

**Review by armfazh — APPROVED** [review]

File: products/time-services/src/content/nts/usage.md [hunk 1]
```diff
@@ -12,5 +12,5 @@ order: 0
 
 If you would like to hear about the development of additional clients,
 updates on our service, or would like to announce that your client
-supports NTS, please email `time-services@cloudflare.com` to be added
-to our distribution list.
+supports NTS, please <a href="https://groups.google.com/a/cloudflare.com/g/time-services">sign up</a>
```
Comment (lukevalenta) [inline products/time-services/src/content/nts/usage.md]: I get "You don't have permission to access this content" at this link, and there's a prompt to email the group's owners and managers. Do we need to update permissions so people can sign up for the group themselves?

**Review by lukevalenta — APPROVED** [review]

File: products/time-services/src/content/nts/usage.md (hunk 1 above)
Comment (wbl) [reply products/time-services/src/content/nts/usage.md]: I fiddled with the settings that should work now.

Comment (lukevalenta) [reply products/time-services/src/content/nts/usage.md]: I confirmed that it now works for me.

Comment (dcpena) [reply products/time-services/src/content/nts/usage.md]: Just wanted to confirm before approving and merging, but I also received a "You don't have permission to access this content" message. Is that the behavior that should occur for all users?

**Comment by wbl** [comment]
On Wed, Nov 17, 2021, 4:19 PM Denise Peña ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In products/time-services/src/content/nts/usage.md
> <https://github.com/cloudflare/cloudflare-docs/pull/2684#discussion_r751644300>
> :
>
> > @@ -12,5 +12,5 @@ order: 0
>
>  If you would like to hear about the development of additional clients,
>  updates on our service, or would like to announce that your client
> -supports NTS, please email ***@***.***` to be added
> -to our distribution list.
> +supports NTS, please <a href="https://groups.google.com/a/cloudflare.com/g/time-services">sign up</a>
>
> Just wanted to confirm before approving and merging, but I also received a
> "You don't have permission to access this content" message. Is that the
> behavior that should occur for all users?
>

No, clearly I need to get the right sign up url from Google groups.

**Comment by kodster28** [comment]
Howdy @lukevalenta, sorry to bug you on this, but I'm also getting the same error from that link. If it's expected behavior, happy to take this and make the required changes to fix the merge conflict.

**Comment by lukevalenta** [comment]
> Howdy @lukevalenta, sorry to bug you on this, but I'm also getting the same error from that link. If it's expected behavior, happy to take this and make the required changes to fix the merge conflict.

Hey @kodster28, thanks for double-checking this! Looks like the group is set to Private right now, so the sign-up link isn't going to work and we'll have to continue manually adding people until we get the group settings changed to Public. Let's leave this PR open for a bit longer until I get that resolved.

**Comment by lukevalenta** [comment]
@kodster28 looks like we don't have a way to make the existing time-services@cloudflare.com mailing list public, and will need to create a separate time-services@cloudflaregroups.com list for that purpose. We can close this PR for now and reopen if we decide to create that list in the future.

**Pull request closed without merging**